model/src/CMakeLists.txt

```diff
@@ -90,12 +96,21 @@ if("OASIS" IN_LIST switches)
 endif()
 
 if(NETCDF)
-  find_package(NetCDF REQUIRED COMPONENTS Fortran)
+  if("netcdf" IN_LIST EXCLUDE_FIND)
+    # NetCDF library flags provided by compiler or wrapper script: don't search
+    set(NetCDF_Fortran_FOUND TRUE)
+	message(WARNING "Not searching for NetCDF library - please ensure correct flags are provided by your compiler/wrapper")
```

Comment:
Just a comment that I think this should be indented in line with `set`.

Reply:
Ah yes - looks like a tab sneaked in there. Fixed now (in upcoming commit)